src/components/pod/pod.test.js

```diff
@@ -13,6 +55,234 @@ const PodWithSoftDelete = () => (
+      (height) => {
+        CypressMountWithProviders(<PodComponent height={height} />);
+        podComponent().should("have.css", "height", height);
+      }
+    );
+
+    it.each([
+      [false, "not.exist"],
+      [true, "be.visible"],
+    ])(
+      "should check when softDelete is %s for Pod component",
+      (boolVal, state) => {
+        CypressMountWithProviders(
+          <SoftDeletePodComponent softDelete={boolVal} />
+        );
+        podSoftDelete().should(state);
+      }
+    );
+
+    it.each([
+      [false, "77.5625px"],
+      [true, "1292px"],
+    ])(
+      "should check when editContentFullWidth is %s for Pod component",
+      (boolVal, width) => {
+        CypressMountWithProviders(
+          <PodComponent editContentFullWidth={boolVal} />
+        );
+        divPodComponent().should("have.css", "width", width);
+      }
+    );
+
+    it.each([true, false])(
+      "should check when displayEditButtonOnHover is %s for Pod component",
+      (boolVal) => {
+        CypressMountWithProviders(
+          <PodComponent displayEditButtonOnHover={boolVal} />
+        );
+        if (boolVal === true) {
+          podContent().click();
+          podEditContainer().should("be.visible");
+          podDelete().should("not.be.visible");
+        } else {
+          podDelete().should("be.visible");
+        }
+      }
+    );
+
+    it.each([
+      [true, "1350px"],
+      [false, "77.5625px"],
+    ])(
+      "should check when internalEditButton is %s for Pod component",
+      (boolVal, width) => {
+        CypressMountWithProviders(
+          <PodComponent internalEditButton={boolVal} />
+        );
+        divPodComponent().should("have.css", "width", width);
+      }
+    );
+  });
+
+  describe("should render Pod component for event tests", () => {
+    let callback;
+
+    beforeEach(() => {
+      callback = cy.stub();
+    });
+
+    it("should call onDelete callback when a click event is triggered for Pod component", () => {
+      CypressMountWithProviders(<PodComponent onDelete={callback} />);
+      podDelete()
+        .click()
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          expect(callback).to.have.been.calledOnce;
+        });
+    });
+
+    it("should call onUndo callback when a click event is triggered for Pod component", () => {
+      CypressMountWithProviders(
+        <SoftDeletePodComponent softDelete onUndo={callback} />
+      );
+      podUndo()
+        .click()
+        .then(() => {
+          // eslint-disable-next-line no-unused-expressions
+          expect(callback).to.have.been.calledOnce;
+        });
+    });
+
+    it.each([false, true])(
+      "should check when triggerEditOnContent is %s for Pod component",
+      (boolVal) => {
+        CypressMountWithProviders(
+          <PodComponent triggerEditOnContent={boolVal} />
+        );
+        if (boolVal === false) {
+          podContent()
```

Comment:
what is the difference between those 2 assertions? and `IF` statement has no sense here